### PR TITLE
Fixed bug in BasicCompletes

### PR DESCRIPTION
**Pull request by zpbappi**

### CODE REVIEW

File: src/Vlingo.Common/Vlingo.Common.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     
     <!-- NuGet Metadata -->
     <IsPackable>true</IsPackable>
-    <PackageVersion>0.3.0</PackageVersion>
+    <PackageVersion>0.3.1</PackageVersion>
```
Comment (tjaskula) [inline src/Vlingo.Common/Vlingo.Common.csproj]: As this is a breaking change it should be rather 0.4, if we align to SemVer

File: src/Vlingo.Common/ICompletes.cs [hunk 2]
```diff
@@ -16,27 +16,27 @@ public interface ICompletes
...
         ICompletes<T> AndThen(Func<T, T> function);
 
-        ICompletes<T> AndThenConsume(long timeout, T failedOutcomeValue, Action<T> consumer);
+        ICompletes<T> AndThenConsume(TimeSpan timeout, T failedOutcomeValue, Action<T> consumer);
         ICompletes<T> AndThenConsume(T failedOutcomeValue, Action<T> consumer);
-        ICompletes<T> AndThenConsume(long timeout, Action<T> consumer);
+        ICompletes<T> AndThenConsume(TimeSpan timeout, Action<T> consumer);
         ICompletes<T> AndThenConsume(Action<T> consumer);
 
-        O AndThenTo<F, O>(long timeout, F failedOutcomeValue, Func<T, O> function);
```
Comment (tjaskula) [inline src/Vlingo.Common/ICompletes.cs]: Why not aligning to .NET conventions `TR` `TF` `TO` ?

**Comment by tjaskula** [comment]
This should fix then #17

File: src/Vlingo.Common/Vlingo.Common.csproj (hunk 1 above)
Comment (zpbappi) [reply src/Vlingo.Common/Vlingo.Common.csproj]: Good point. Will do.

File: src/Vlingo.Common/ICompletes.cs (hunk 2 above)
Comment (zpbappi) [reply src/Vlingo.Common/ICompletes.cs]: done.

**Pull request merged**